src/minecraft/handlers/ChatHandler.js

```diff
@@ -209,19 +209,28 @@ class StateHandler extends EventHandler {
           let user = message.split(' ')[1]
           user = user.replace('-----------------------------------------------------', '')
           user = user.replace('\n', '')
-          if (config.discord.autoAccept===true){
-            this.bot.chat(`/g accept ${user}`)
-          }
+          if (config.discord.autoAccept===true){  
+            const blacklistedPlayers = config.discord.blacklistedPlayers
+            if (blacklistedPlayers.includes(user)) {
+              return;
+            } else {
+              this.bot.chat(`/g accept ${user}`)
+            }
           return this.minecraft.broadcastCleanEmbed({ message: `${user} has requested to join the guild.`, color: '47F049' })
```

Comment:
we should broadcast the join message anyways
```suggestion
          if (config.discord.autoAccept===true){  
            let blacklistedPlayers = config.discord.blacklistedPlayers
            if (!blacklistedPlayers.includes(user)) {
              this.bot.chat(`/g accept ${user}`)
            }
          return this.minecraft.broadcastCleanEmbed({ message: `${user} has requested to join the guild.`, color: '47F049' })
```